py/torch_tensorrt/dynamo/_exporter.py

```diff
@@ -45,19 +43,11 @@ def export(
                         format=torch.channel_last
                     ), # Dynamic input shape for input #2
                     torch.randn((1, 3, 224, 244)) # Use an example tensor and let torch_tensorrt infer settings
-        ir (str): torchscript | exported_program. Based on the provided ir, the output type would be a torchscript or exported program.
     """
-    if ir == "torchscript":
-        return torch.jit.trace(gm, inputs)
```

Comment:
What are the options for users who want to `torchscript` the exported product for serialization?

Reply:
They could always post return trace

Reply:
Alternatively, we can have a string/enum, input for `output_format` with options: `exported_program/ep`, `torchscript/ts`, `graph_module/fx`

Reply:
^ This looks good. Added `output_format` now